packages/web/docs/src/components/large-callout.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { ReactNode } from 'react';
+import { cn } from '@theguild/components';
+
+export interface LargeCalloutProps extends React.HTMLAttributes<HTMLDivElement> {
+  icon?: ReactNode;
+  variant: 'primary' | 'secondary';
+  heading?: string;
+  cta?: ReactNode;
+}
+
+export function LargeCallout({
```

Comment:
Named like this, because Nextra already has a `Callout` and I ain't gonna migrate all of that for sure.